### PR TITLE
turn off ATMs

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
ATMs are still working. That's odd
#### Describe the solution
move ATM from terrain to furniture
update it's deconstuction and bash drops
make a copy that is turned off
replace all vanilla instances (and one in no hope) of powered one with off version
#### Testing
image
image

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture_misc.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        { "item": "money_five", "count": [ 0, 299 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_atm_off",
+    "//": "www.testlinkusa.com/ncr-5870",
+    "name": "ATM",
+    "description": "Design to withdraw money from your bank accoun, this this is completely dead now, both from having no more power, and having no internet connection to do any withdraws.  While you can deconstruct it, it's effectively just a big pile of metal.",
```
Comment (Karol1223) [inline data/json/furniture_and_terrain/furniture_misc.json:66]:
```suggestion
    "description": "Designed to withdraw money from your bank account, this machine is completely dead now, both from having no more power, and having no internet connection to do any withdraws.  While you can deconstruct it, it's effectively just a big pile of metal.",
```

**Review by Karol1223 — CHANGES_REQUESTED** [review]

**Comment by x-qq** [comment]
How do you load currency bundles onto your cash card now after this?

**Comment by GuardianDll** [comment]
you don't, and you shouldn't

**Comment by x-qq** [comment]
I think it's a fun feature and at least one working ATM somewhere should be kept operational.

**Comment by GuardianDll** [comment]
only in aftershock, with working internet connection

**Comment by x-qq** [comment]
An ATM inside a bank building will not need internet, it is connected directly into bank network, and can stay powered for a very long time by bank's backup genrator (practically indefinitely assuming advanced enough technology).

**Comment by GuardianDll** [comment]
> connected directly into bank network

what makes bank network working? what holds servers of the bank miles away still online? grid is dead in the whole country

**Comment by x-qq** [comment]
The bank server is inside the bank building. The ATM is connected to it by cable. It is all powered by backup power.

**Comment by harakka** [comment]
> can stay powered for a very long time by bank's backup genrator (practically indefinitely assuming advanced enough technology).

We don't have any particularly advanced technology. A random bank office doesn't have backup power to keep it running indefinitely or indeed for any notable length of time.

**Comment by Kozakow** [comment]
As of right now, cash cards are only useful for vending machines and gas stations, former of which can be relatively easily broken into. Since the ammo vending machines received the SHIFT+DELETE treatment, there really is no other use for electronic money (robot helper is just a gimmick, and maybe a good source of electronic parts). 

Since gas stations working are also a grey area, i doubt there really is any reason to keep inventing excuses for why ATMs should be operational. Once there is a way to siphon fuel from the underground tanks (or those located on the surface, which our 15 INT post-treshold Alpha mutant survivor still can't get into), cash cards as a whole will lose all purpose.

**Comment by Little-119** [comment]
>Once there is a way to siphon fuel from the underground tanks (or those located on the surface, which our 15 INT post-treshold Alpha mutant survivor still can't get into), cash cards as a whole will lose all purpose.

That's not true!
...they'll still be usable as arrow fletching.

**Pull request merged**